apps/src/webapp/webapp.js

```diff
@@ -1556,6 +1597,32 @@ Webapp.clearTimeout = function (opts) {
   window.clearTimeout(opts.timeoutId);
 };
 
+Webapp.createRecord = function (opts) {
+  var record = codegen.marshalInterpreterToNative(opts.record);
+  FormStorage.createRecord(record,
+      Webapp.handleCreateRecord.bind(this, opts.callback));
```

Comment:
any synchronous return value needed for success/failure? (applies to this and readRecords below)